integration_test/exqlite/test_helper.exs

```diff
@@ -41,9 +56,15 @@ _ = Ecto.Adapters.Exqlite.storage_down(TestRepo.config())
 :ok = Ecto.Adapters.Exqlite.storage_up(TestRepo.config())
 
 {:ok, _} = TestRepo.start_link()
+{:ok, _pid} = PoolRepo.start_link()
 
 :ok = Ecto.Migrator.up(TestRepo, 0, Ecto.Integration.Migration, log: false)
 Ecto.Adapters.SQL.Sandbox.mode(TestRepo, :manual)
 Process.flag(:trap_exit, true)
 
-ExUnit.start()
+ExUnit.start(
+  exclude: [
+    # SQLite does not have an array type
+    :array_type,
```

Comment:
It doesn't have an array type, but I specified it to get serialized as a json array. Not sure if this is just for the postgres specific array operation stuff.

Reply:
Yea there is no way we can support that. Unfortunate.